learningresources/tests/test_api.py

```diff
@@ -61,6 +64,10 @@ def test_create(self):
             create_course(**self.kwargs)
         self.assertTrue(course_count() == after)
 
+        # NOT NULL constraint fails on org
+        with self.assertRaises(IntegrityError):
```

Comment:
If this line:

```python
with self.assertRaises(IntegrityError):
```

is changed to this:

```python
with self.assertRaises(IntegrityError) as ex:
```

it would become possible to check that the error was caused by the org field.